addons/exact-count/addon.json

```
@@ -15,6 +15,10 @@
     }
   ],
   "info": [
+    {
+      "id": "studioClick",
+      "text": "To see the exact number of projects in a studio, click the \"100+\" on the Projects tab. The \"exact count for studios\" setting below needs to be enabled."
```

```suggestion
      "text": "To see the exact number of projects in a studio, click the \"100+\" label on the Projects tab. The \"exact count for studios\" setting below also needs to be enabled."
```

---

To me, "also needs to be enabled" sounds like this setting isn't the only thing that needs to be enabled. I think the following might be better:
```suggestion
      "text": "To see the exact number of projects in a studio, enable the \"exact count for studios\" setting below and click the \"100+\" label on the studio's Projects tab."
```